### PR TITLE
Task/auth support subscriptions

### LINKED ISSUE #592: missing support for authentication for NGSIv1 subscriptions
current code does not provide any authentication means for subscriptions

### PR DESCRIPTION
**Pull request by chicco785**
this pull request fixes #592
it should be rebased after base oauth2 support is merged (#591)

### CODE REVIEW

**Comment by fgalan** [comment]
Now that PR #591 has been merged, this one can be rebased.

**Comment by chicco785** [comment]
rebased

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -6,3 +6,4 @@
 - FIX Correctly return 404 when asking to remove a not existing device (#585)
 - Add: support HTTPS requests toward Keystone (if url is used in the authorization configuration)
 - Add: support OAuth2 provider as authentication backend (#591)
+- Add: support for authentication to subscription requests (#600)
```
[the remark this answers is not in the document]
Comment (chicco785) [reply CHANGES_NEXT_RELEASE]: fix by 7513d36

**Comment by fgalan** [comment]
Overpassed by PR #764. Closing.

**Pull request closed without merging**